src/main/java/frc/robot/subsystems/ControlPanel.java

```diff
@@ -81,21 +81,21 @@ public ColorStates getTargetColor() {
         gameData = DriverStation.getInstance().getGameSpecificMessage();
         if (gameData.length() > 0) {
             switch (gameData.charAt(0)) {
-            case 'R':
-                color = ColorStates.BLUE;
-                break;
-            case 'Y':
-                color = ColorStates.GREEN;
-                break;
-            case 'B':
-                color = ColorStates.RED;
-                break;
-            case 'G':
-                color = ColorStates.YELLOW;
-                break;
-            default:
-                color = ColorStates.OTHER;
-                break;
+                case 'R':
```

Comment:
Indentation changes?

Reply:
pretty sure that was autoformat- should i leave it?

Reply:
Meh fine